src/characters/tDec.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  EncryptedTreasureMap,
+  PublicKey,
+  SecretKey,
+} from '@nucypher/nucypher-core';
+
+import { fromHexString } from '../utils';
+
+import { Enrico } from './enrico';
+import { tDecDecrypter } from './universal_bob';
+
+const tDecConfig: { [key: string]: { [key: string]: any } } = {
```

Comment:
We definitely want to move to a more robust solution. We've had a couple of brief discussions (https://discord.com/channels/411401661714792449/469956126709121026/993575795815612576). The current thinking is that we'll have a separate repo that contains the different scheme configurations.

For now I think this ok, because it is only on `ibex`

Reply:
Ah yes - makes sense 👍

Reply:
this is now done